build_rln.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# This script is used to build the rln library for the current platform, or download it from the
+# release page if it is available.
+
+set -e
+
+# first argument is the build directory
+build_dir=$1
+
+if [ -z "$build_dir" ]; then
+    echo "No build directory specified"
+    exit 1
+fi
+
+# Get the host triplet
+host_triplet=$(rustup show | grep "Default host: " | cut -d' ' -f3)
+
+# Download the prebuilt rln library if it is available
+if wget https://github.com/vacp2p/zerokit/releases/download/nightly/$host_triplet-rln.tar.gz
```

Comment:
```suggestion
if wget https://github.com/vacp2p/zerokit/releases/download/v0.2/$host_triplet-rln.tar.gz
```
note: we will use a static version some time in the future, keeping nightly in for now will allow us to detect breaking changes faster